apps/console/app/routes/_shell/sidebar-nav.tsx

```
@@ -29,23 +33,24 @@ type SidebarNavProps = {
 export const SidebarNav = ({ activeAgent, activeBranch }: SidebarNavProps) => {
   const { pathname } = useLocation();
   const navigate = useNavigate();
+  const basePath = `/agent/${activeAgent.slug}/branch/${activeBranch.slug}`;
 
-  navItems.forEach(({ shortcut, getPath }) => {
+  navItems.forEach(({ shortcut, postfix }) => {
     useHotkeys(
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)